deploy/rosa-oauth-templates-policies/50-rosa-oauth-tpl-errors.Policy.yaml

```diff
@@ -6,16 +6,16 @@ metadata:
         policy.open-cluster-management.io/categories: CM Configuration Management
         policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
         policy.open-cluster-management.io/standards: NIST SP 800-53
-    name: rosa-oauth-templates-errors
-    namespace: openshift-rosa-oauth-templates-policies
+    name: rosa-oauth-tpl-errors
+    namespace: openshift-rosa-oauth-tpl-policies
```

Comment:
Hello, the namespace on line 42 and line 55 (in the Placementrule and Placementbinding section) need to be changed as well !